packages/api/src/routes/internal/medical/patient-settings.ts

```diff
@@ -75,4 +76,23 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /internal/patient/settings/adt-subscribers
+ *
+ * Gets all patients that have ADT subscriptions enabled for the given states.
+ *
+ * @param req.query.states List of US state codes to filter by
+ * @returns List of patients with ADT subscriptions in the specified states
+ */
+router.get(
+  "/adt-subscribers",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const states = getFromQueryAsArrayOrFail("states", req);
+    const result = await getAdtSubscribers(states);
+
+    return res.status(status.OK).json(result);
+  })
+);
```

Comment:
this endpoint needs to be paginated, right? I think we'll be returning like 40K patients for texas here.